File: CS/RegisterFromLogonFormSolution.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;


namespace RegisterFromLogonFormSolution.Module {
    public sealed partial class RegisterFromLogonFormSolutionModule : ModuleBase {
        public RegisterFromLogonFormSolutionModule() {
            InitializeComponent();
        }
        protected override IEnumerable<Type> GetDeclaredExportedTypes() {
            List<Type> result = new List<Type>(base.GetDeclaredExportedTypes());
            result.Add(typeof(RegisteredUser));
            return result;
        }
    }
}

File: VB/RegisterFromLogonFormSolution.Module/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Imports DevExpress.ExpressApp
Imports System.Reflection


Namespace RegisterFromLogonFormSolution.Module
	Public NotInheritable Partial Class RegisterFromLogonFormSolutionModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Protected Overrides Function GetDeclaredExportedTypes() As IEnumerable(Of Type)
			Dim result As New List(Of Type)(MyBase.GetDeclaredExportedTypes())
			result.Add(GetType(RegisteredUser))
			Return result
		End Function
	End Class
End Namespace
